src/invidious/yt_backend/extractors.cr

```diff
@@ -585,14 +585,14 @@ private module Parsers
   #
   module ContinuationItemRendererParser
     def self.process(item : JSON::Any, author_fallback : AuthorFallback)
-      if item_contents = item["continuationItemRenderer"]?
+      if item_contents = (item.dig?("continuationItemRenderer", "continuationEndpoint") || item.dig?("continuationEndpoint"))
```

Comment:
If I'm not mistaken since you're just passing in `node` here

https://github.com/iv-org/invidious/blob/3c6742be50d40fb1e393d4ad731c673530ab4181/src/invidious/yt_backend/extractors.cr#L724-L725

I don't believe the changes to `ContinuationItemRendererParser` is necessary